### ISSUE TITLE
PS4 version displays outside of system DISPLAY SETTINGS area

### ISSUE THREAD

**Issue by iamamitten** [Cpasjuste/pplay#75]
attempting to open OPTIONS does nothing, I can look at half of the left bar menu and choose between HOME and EXIT that actually do anything

**Comment by Cpasjuste** [comment]
Hi,

Are you using a 720p resolution? If so, it's not handled yet, only 1080p.

**Comment by iamamitten** [comment]
No, I'm using 1080p, my TV has overscan that can't be disabled.

**Comment by neo305** [comment]
Same here I can’t see the menu so no idea how to fix this

**Comment by reza-yb** [comment]
Same problem here

**Comment by HackZy01** [comment]
My Options tab doesn't even work

**Comment by Immortallix** [comment]
Same problem here, on a 720p TV. I can't even see the UI
